examples/tensorflow2/tensorflow2_mlflow.ipynb

```diff
@@ -0,0 +1,753 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Parse model response to find the highest prediction probability."
```

Comment:
In the following cells, we will parse the model response to find the highest prediction probability.